tests/phpunit/tests/rest-api/wpRestTemplateRevisionsController.php

```diff
@@ -0,0 +1,276 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Template_Revisions_Controller functionality.
+ *
+ * @package WordPress
+ * @subpackage REST API
+ *
+ * @group restapi
+ */
+class Tests_REST_wpRestTemplateRevisionsController extends WP_Test_REST_Controller_Testcase {
+
+	const TEST_THEME    = 'block-theme';
+	const TEMPLATE_NAME = 'my_template';
+
+	/**
+	 * Admin user ID.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var int
+	 */
+	private static $admin_id;
+
+	/**
+	 * Template post.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var WP_post
+	 */
+	private static $template_post;
+
+	/**
+	 * Create fake data before our tests run.
+	 *
+	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
+	 */
+	public static function wpSetUpBeforeClass( $factory ) {
+
+		self::$admin_id = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+		wp_set_current_user( self::$admin_id );
+
+		// Set up template post.
+		self::$template_post = $factory->post->create_and_get(
+			array(
+				'post_type'    => 'wp_template',
+				'post_name'    => self::TEMPLATE_NAME,
+				'post_title'   => 'My Template',
+				'post_content' => 'Content',
+				'post_excerpt' => 'Description of my template',
+				'tax_input'    => array(
+					'wp_theme' => array(
+						self::TEST_THEME,
+					),
+				),
+			)
+		);
+		wp_set_post_terms( self::$template_post->ID, self::TEST_THEME, 'wp_theme' );
+
+		// Update post to create a new revisions.
+		$updated_template_post = array(
+			'ID'           => self::$template_post->ID,
+			'post_content' => 'Content revision #2',
+		);
+
+		wp_update_post( $updated_template_post, true, false );
+
+		// Update post to create a new revisions.
+		$updated_template_post = array(
+			'ID'           => self::$template_post->ID,
+			'post_content' => 'Content revision #3',
+		);
+
+		wp_update_post( $updated_template_post, true, false );
+
+		// Update post to create a new revisions.
+		$updated_template_post = array(
+			'ID'           => self::$template_post->ID,
+			'post_content' => 'Content revision #4',
+		);
+
+		wp_update_post( $updated_template_post, true, false );
+
+		// Update post to create a new revisions.
+		$updated_template_post = array(
+			'ID'           => self::$template_post->ID,
+			'post_content' => 'Content revision #5',
+		);
+
+		wp_update_post( $updated_template_post, true, false );
+	}
+
+	public function set_up() {
+		parent::set_up();
+	}
```

Comment:
Not needed.

Reply:
Absolutely.

The version you reviewed was a work in progress. That method has been removed since your last review.